en/docs/learn/api-controller/getting-started-with-wso2-api-controller.md

```diff
@@ -427,7 +427,7 @@ Follow the instructions below to display a list of APIs/API Products/Application
                 apictl list apis --environment production --limit 15 --insecure
                 ```    
                 ```go
-                apictl list apis --environment production --query provider:Alice name:PizzaShackAPI --insecure
+                apictl list apis --environment production --query provider:Alice name:PizzaShackAPI version:2.0.0 --insecure
```

Comment:
Shouldn't we have quotations as below?
--query "provider:Alice name:PizzaShackAPI version:2.0.0"

Reply:
@wasuradananjith seems like quotations are not necessary to execute this command with query params.
![Screenshot from 2021-07-06 12-40-28](https://user-images.githubusercontent.com/42435576/124557860-9c41a180-de57-11eb-9614-cf6526574fbe.png)

Reply:
Thanks. Ack